### ISSUE TITLE
Please add support for qualified identifiers

### ISSUE THREAD

**Issue by axos88** [Zerounary/struct-convert#9]
```
use api::Quota as ApiQuota;

#[derive(Convert)]
#[convert(from = "ApiQuota")] // Works
#[convert(into = "ApiQuota")] // Works
#[convert(from = "api::Quota")] // Panics: message: `"api::Quota"` is not a valid identifier
#[convert(into = "api::Quota")] // Panics: message: `"api::Quota"` is not a valid identifier
pub struct Quota {
  ...
}
```

**Comment by axos88** [comment]
#4 might be related, or even duplicate

**Comment by axos88** [comment]
Comfirmed that it's working on rev 92bff112a848bb371954cd0f66d7447bb5cfaa7b, but it hasn't been released to the repository as v1.2.8. Can you please do that?

**Comment by axos88** [comment]
So #8 is related as well :)

**Comment by Zerounary** [comment]
Sorry, I have now released version 1.2.8 on crites.io
